x-pack/plugins/security_solution/server/lib/timeline/utils/common.ts

```diff
@@ -39,6 +39,16 @@ export const buildFrameworkRequest = async (
 
 export const escapeHatch = schema.object({}, { unknowns: 'allow' });
 
+export const getNotesPaginated = schema.object({
```

Comment:
Can you update the schema.yml as well for the notes route to include these? Thx!